docs/sources/csv.md

```diff
@@ -97,29 +97,29 @@ Germany,700
 
 ## Columns
 
-Though your csv file have columns, specify them as columns manually and only that fields will be considered for display. Columns will appear in the same order you specify. Each column will have following properties
+Even if your CSV file has columns, you should specify them manually, and only those fields will be considered for display. The columns will appear in the order you specify. Each column will have the following properties:
 
-| Column   | Description                                     |
-| -------- | ----------------------------------------------- |
-| Title    | Title of the column when using the table format |
-| Selector | Column name in CSV file. Case sensitive         |
-| Format   | Format of the column                            |
+| Column   | Description                                      |
+| -------- | -----------------------------------------------  |
+| Title    | Title of the column when using the table format. |
+| Selector | Column name in CSV file. Case sensitive.         |
+| Format   | Format of the column.                            |
 
-If you don't specify any columns, then the infinity plugin will try to auto generate the columns and all the fields will be returned as string. (This auto generate columns feature only works with table format)
+If you don't specify any columns, then the Infinity plugin will try to auto generate the columns and all the fields will be returned as string. Please note that this auto generate columns feature only works with table format.
 
 ## CSV URL
 
-In the below example, we are going to convert the CSV URL `https://gist.githubusercontent.com/yesoreyeram/64a46b02f0bf87cb527d6270dd84ea47/raw/32ae9b1a4a0183dceb3596226b818c8f428193af/sample-with-quotes.csv` into a grafana datasource.
+In the example below, we are going to convert the CSV URL `https://gist.githubusercontent.com/yesoreyeram/64a46b02f0bf87cb527d6270dd84ea47/raw/32ae9b1a4a0183dceb3596226b818c8f428193af/sample-with-quotes.csv` into a Grafana data source.
```

Comment:
https://developers.google.com/style/word-list#below

We should also avoid using "we" in this way but that can be considered out of scope for this patch: https://developers.google.com/style/person

```suggestion
In the following example, we are going to convert the CSV URL `https://gist.githubusercontent.com/yesoreyeram/64a46b02f0bf87cb527d6270dd84ea47/raw/32ae9b1a4a0183dceb3596226b818c8f428193af/sample-with-quotes.csv` into a Grafana data source.
```